Cabal/doc/developing-packages.rst

```diff
@@ -981,17 +981,28 @@ describe the package as a whole:
 
     A limited form of ``*`` wildcards in file names, for example
     ``data-files: images/*.png`` matches all the ``.png`` files in the
-    ``images`` directory.
-
-    The limitation is that ``*`` wildcards are only allowed in place of
-    the file name, not in the directory name or file extension. In
-    particular, wildcards do not include directories contents
-    recursively. Furthermore, if a wildcard is used it must be used with
-    an extension, so ``data-files: data/*`` is not allowed. When
-    matching a wildcard plus extension, a file's full extension must
-    match exactly, so ``*.gz`` matches ``foo.gz`` but not
-    ``foo.tar.gz``. A wildcard that does not match any files is an
-    error.
+    ``images`` directory. ``data-files: audio/**/*.mp3`` matches all
```

Comment:
the documentation should mention that `**` is available starting with `cabal-version:3.0` (if we assume that 3.0 will be the next major release)

Reply:
Done. Also went ahead and changed all the '2.4' mentions to '3.0'.